editor/editor_node.cpp

```diff
@@ -2476,13 +2476,9 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 						file->set_current_path(path.replacen("." + ext, "." + extensions.front()->get()));
 					}
 				}
-			} else {
-				String existing;
-				if (extensions.size()) {
-					String root_name(scene->get_name());
-					existing = root_name + "." + extensions.front()->get().to_lower();
-				}
-				file->set_current_path(existing);
```

Comment:
This call will be ignored if `String` is empty, so I simplified the check a bit.